webrtc.html

```diff
@@ -1873,28 +1800,100 @@ <h4>Set the RTCSessionDescription</h4>
+                            to <code>null</code>.</p>
+                          </li>
+                          <li>
+                            <p>Set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiveCodecs]]</a>
+                            to <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            and set <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\StableStateReceiveCodecs]]</a>
+                            to an empty list.</p>
+                          </li>
+                          <li data-tests="RTCPeerConnection-setDescription-transceiver.html">
+                            <p>If the <var>transceiver</var> was created by applying the
+                            <code><a>RTCSessionDescription</a></code> that is
+                            being rolled back, and a track has never been attached to
+                            it via <code>addTrack</code>, run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p><a data-lt="stop the RTCRtpTransceiver">Stop the
+                                RTCRtpTransceiver</a> <var>transceiver</var>.</p>
+                              </li>
+                              <li>
+                                <p>Remove <var>transceiver</var> from <var>connection</var>'s
+                                <a href="#transceivers-set">set of transceivers</a>.</p>
+                              </li>
+                              <li>
+                                <p><a>Set the associated remote streams</a> given
+                                <var>transceiver</var>.<a>[[\Receiver]]</a>, <var>msids</var>
+                                as the empty list, <var>addList</var>, and <var>removeList</var>.</p>
+                              </li>
+                              <li>
+                                <p>If the <var>transceiver</var>'s <a>[[\FiredDirection]]</a>
+                                is either <code>"sendrecv"</code> or <code>"recvonly"</code>,
+                                <a>process the removal of a remote track</a> given
+                                <var>transceiver</var> and <var>muteTracks</var>.</p>
+                              </li>
+                            </ol>
+                          </li>
+                          <li>
+                            <p>Otherwise, (if the <var>transceiver</var> was not just removed)
+                            run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p>If the <var>transceiver</var>'s <a>[[\FiredDirection]]</a>
+                                is either <code>"sendonly"</code> or <code>"inactive"</code>
+                                and <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> is
+                                either <code>"sendrecv"</code> or <code>"recvonly"</code>,
```

Comment:
Is "the previous step" referring to the "set the associated remote streams" step that occurred due to rollback?

I added:

> or the "set the associated remote streams" step above increased the length of <var>addList</var>

But note that this is now referring to a step that is above the "If the transceiver was created by applying... Otherwise, (if the transceiver was not just removed)..." steps. See if that is OK or if I should move the "set the associated remote streams" step to here to make this more clear (in which case I also have to run the "set the associated remote streams" step for the transciever-being-removed clause again).